Configurations.md

```diff
@@ -2353,7 +2353,7 @@ Require a specific version of rustfmt. If you want to make sure that the
 specific version of rustfmt is used in your CI, use this option.
 
 - **Default value**: `CARGO_PKG_VERSION`
-- **Possible values**: any published version (e.g. `"0.3.8"`)
+- **Possible values**: `semver` compliant values, such as defined on [semver.org](https://semver.org/). Multiple values can be used, they should be separeted by a comma, e.g.: `1.0.0, 1.1.0`. By default, values without comparison operators are treated as `^` (caret) version requirements. If you want to specify an exact version, use the operator `=`, e.g.: `=1.0.0`.
 - **Stable**: No (tracking issue: [#3386](https://github.com/rust-lang/rustfmt/issues/3386))
 
```

Comment:
I think the first sentence is good and can stay here for the  `Possible values`, but I think we should move the rest of the explanation to a small paragraph below. Maybe even a bulleted list showing users how to target exact, major, minor, and patch versions. Oh and maybe a range of versions too like you mentioned here https://github.com/rust-lang/rustfmt/issues/6063#issue-2125707100.